app/facades/food_data_facade.rb

```diff
@@ -0,0 +1,14 @@
+class FoodDataFacade
+
+  def self.foods(query)
+    food_details = FoodDataService.search_foods(query)[:foods][0..9]
+ 
+    food_details.map do |food|
+      FoodDataDetails.new(food)
+    end
+  end
+
+  def self.total_results(query)
```

Comment:
It would be best if we only had to hit the service once. You could combine these methods and save the totalResults value to a variable and pass both the food objects and the totalHits value back to the controller as either a Hash or an Array that you could  then deconstruct into separate instance variables. This would allow you to only hit the external API once. Faster code!